src/Database.cs

```diff
@@ -0,0 +1,237 @@
+using System;
+using System.Collections.Frozen;
+using System.Collections.Generic;
+using System.Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace NotifierRedirecter;
+
+public sealed class Database
+{
+    // Sqlite is thread safe by default, meaning it's safe to use the same connection across multiple threads: https://stackoverflow.com/a/39916593/10942966
+    private readonly SqliteConnection _connection;
```

Comment:
Why use `SqliteConnection` instead of `SqliteFactory`?

Reply:
Because we're only using a single connection. The connection itself is thread safe and using a factory would not provide any benefits that the connection doesn't already provide.

Reply:
Ah ic